01-Data-Ecosystem/Chapter_1.4_Columns_and_Variables.ipynb

```diff
@@ -343,7 +343,7 @@
         "colab_type": "text"
       },
       "source": [
-        "1\\. Read in the Titanic data set. Identify each variable in the Titanic data set as either quantitative, categorical, or other. Cast all variables to the right type and assign them back to the `DataFrame`."
+        "1\\. Read in the Titanic data set. Identify each variable in the Titanic data set as either quantitative, categorical, or other. Cast all variables to the right type and assign them back to the `DataFrame`. *Note: deciding the appropriate type can be tricky! Think carefully about the `'name'`, `'ticketno'`, and `'survived'` columns, in particular.*"
```

Comment:
This is something students had trouble with. My guess is that the following are the intended answers:
- `'name'`: other. Even though there are a handful of collisions in the data set, there are too many unique values to do meaningful analysis as a categorical variable.
- `'ticketno'`: categorical. Although the values happen to be numbers, I don't think we can meaningfully treat the ticket number as interval might as well be arbitrary symbols for which equality comparison is defined. However, there are a limited set of a values, and one can do meaningful analysis treating it as a categorical variable (e.g. `groupby` to see who booked together).
- `'survived'`: categorical.

 I'm not sure if this is the right text to add, but I thought it was worth discussion, so I put in a placeholder.